### PR TITLE
docs: add WordPress limitation section (#303)

### PR DESCRIPTION
**Pull request by matzeeable**
See #303

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ potentially very difficult to debug due to dissimilar or unsupported package ver
     - [Composer](#composer-autoloader)
     - [Composer Plugins](#composer-plugins)
     - [PSR-0 Partial support](#psr-0-partial-support)
+    - [WordPress](#wordpress)
```
Comment (theofidry) [inline README.md:68]: Is it a limitation though?

I realise there is a mistake above in the ToC: Phive, PHAR and Composer should be under "installation" not "usage".  And maybe you can create a new ToC entry under "usage", e.g. "Wordpress integration"

File: README.md [hunk 2]
```diff
@@ -873,6 +874,13 @@ transforming it to PSR-4, i.e. in the case above:
 
 If this will work for the classes under `src/JsonMapper/`, it will not for `JsonMapper.php`.
 
+### WordPress
+
+Currently, PHP-Scoper is built in such a way that it generates a completely enclosed application. However, since this does not work in plugin landscapes, because dependencies of not bundled functions and classes are needed, there is currently no optimal solution within PHP-Scoper for WordPress.
+
+There is an open issue regarding [better support of WordPress API](https://github.com/humbug/php-scoper/issues/303). If you agree to migrate your plugin or want to start a new plugin you can use [**WP React Starter**](https://github.com/devowlio/wp-react-starter) - it's an open source boilerplate for WordPress plugins. **Why?** Well, WP React Starter its mission is to be the most modern and best boilerplate in the WordPress community and now PHP-Scoper is part of it.
+
+If you are not able to migrate you can have a look at the solution itself, read more about it here [#303 (comment)](https://github.com/humbug/php-scoper/issues/303#issuecomment-614207305).
```
Comment (theofidry) [inline README.md:883]: I would prefer to keep all links at the bottom (see the existing ones under Credits)

Comment (theofidry) [inline README.md:883]:
```suggestion
As of now PHP-Scoper does not easily allow to completely leave unchanged non-included third-party code causing some hurdles with Wordpress plugins. There is a standing issue about providing a better integration out of the box (See [#303](#303)) but alternatively you can already use [**WP React Starter**][wp-react-starter] which is a boilerplate for WordPress plugins.
```

**Comment by theofidry** [comment]
@matzeeable I've committed some changes. Could you review it and see if it's good for you? Also let me know if you feel there is something missing that you want to convey

**Comment by matzeeable** [comment]
Sorry, I was a bit busy and would have planned some changes for tonight - but now that you have solved it perfectly, everything is fine! Thanks for your effort! Btw. I did not know that you can also list links at the bottom of Markdown.

**Comment by dugajean** [comment]
This PR looks good to go.

**Comment by matzeeable** [comment]
@theofidry I think you can merge this PR :-)

**Comment by theofidry** [comment]
Thanks for the pings :)

**Pull request merged**